test/unit/resources/replication_agent_spec.rb

```diff
@@ -40,9 +45,17 @@
         transport_user: 'someuser',
         transport_password: 'somepassword',
         log_level: 'info',
-        retry_delay: 60_000
+        retry_delay: 60_000,
+        serialization_type: 'durbo',
+        user_id: '',
+        enabled: 'true',
+        ssl: 'relaxed',
+        http_expired: nil
```

Comment:
Does having the value `nil` translate to not passing parameter to the corresponding AEM endpoint? If so, then I think having default value of `nil` would be the correct thing to do instead of passing empty string ``, please replace the default values with nils instead of empty strings.

Unless there's a good reasoning behind passing empty string.
Prior to this PR, the new params didn't exist and hence never constructed in the request params sent to AEM endpoint. My concern is that default to empty string will modify the request to have the params with no value instead of no param at all.